### DISCUSSION TITLE
R2S_v23.05.3_0418 无法运行

### DISCUSSION THREAD

**Discussion by bycolt** [nicholas-opensource/OpenWrt-Autobuild#43] [General]
这个版本启动后能运行大概 10 秒钟就挂掉了，时间短到还没登录进 LuCI 界面，只能通过 ping 192.168.1.1 知道运行了一下然后就 ping 不通了，也尝试过重刷好几次固件都是同样的问题。

**Comment by KuGouGo** [comment]
x86有同样的情况

**Comment by nicholas-opensource** [comment]
网卡驱动问题，我会回滚的

**Comment by nicholas-opensource** [comment]
Fixed in https://github.com/nicholas-opensource/OpenWrt-Autobuild/releases/tag/3.3-re1
